### DISCUSSION TITLE
Implementing a Background Image props

### DISCUSSION THREAD

**Discussion by danielefinocchiaro** [xyflow/xyflow#3944] [Ideas]
It might be useful to implement a property to pass to the Background component that renders a background image with a scale. The image then follows the viewport, zooms in and zooms out. (Useful for building floor plans of homes with a view from the top, draggable furniture as custom blocks).

SUPPPERR COOOOL!!

**Comment by moklick** [comment]
You would do this with a custom node. It could have the lowest z-index and no user interaction.
